.github/workflows/publish-docker-images.yaml

```diff
@@ -0,0 +1,94 @@
+name: Publish Docker Images
+
+on:
+  # `push` will build and public the image with a tag corresponding to the branch name (such as `main`, `master`).
+  # `push tag` will build and public the image with a tag has the same name as git tag (such as `v1.2.3`, `v0.0.1-alpha.1`) and a tag named `latest`.
+  push:
+    branches: [main, master]
+    tags: [v*]
+  # `pull request` will only build to test the image, not publish.
+  pull_request:
+    # The branches below must be a subset of the branches above.
+    branches: [main, master]
+  # `release publish` is almost the same as `push tag`.
+  release:
+    types: [published]
+
+jobs:
+  publish_copilot_proxy:
+    name: Publish copilot_proxy
+    # The `-latest` runner images are the latest **stable** images that GitHub provides.
+    # Here it is equivalent to `ubuntu-20.04`.
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Extract Docker metadata
+        uses: docker/metadata-action@v4
+        with:
+          # define image name
+          images: moyix/copilot_proxy
```

Comment:
@moyix do you want to keep using your personal account or shall we create one for FauxPilot?